code/game/machinery/vending.dm

```diff
@@ -956,10 +956,12 @@
 	icon_vend = "sec-vend"
 	vend_delay = 14
 	req_access = list(access_security)
-	products = list(/obj/item/weapon/handcuffs = 8,/obj/item/weapon/grenade/flashbang = 4,/obj/item/weapon/grenade/chem_grenade/teargas = 4,/obj/item/device/flash = 5,
+	products = list(/obj/item/weapon/handcuffs = 8,/obj/item/weapon/grenade/chem_grenade/teargas = 4,/obj/item/device/flash = 5,
```

Comment:
probably best to remove flashes and tear gas here too, and add them to the comment

Reply:
Will do.

Reply:
After consideration, flashes won't matter. They still spawn all over the ship. Removing from the secvend won't do much, plus, flashes do not stun outright I believe, and someone has to get way close.